examples/detector/ant.py

```diff
@@ -1,5 +1,7 @@
 from pycbc.detector import Detector
 from pycbc.waveform import get_td_waveform
+from matplotlib import pyplot as plt
+import numpy as np
```

Comment:
Might be missing something, but don't think these new imports are used??

Reply:
Woops, good catch. I had experimented with adding some plots, but ending up not adding them this time.